concordium-consensus/src/Concordium/KonsensusV1/Scheduler.hs

```diff
@@ -235,7 +235,7 @@ doEpochTransition True epochDuration theState0 = do
                     applyPendingChanges av (<= epochEnd)
                         <$> bsoGetActiveBakersAndDelegators theState7
                 let BakerStakesAndCapital{..} =
-                        computeBakerStakesAndCapital
+                        computeBakerStakesAndCapital @m
```

Comment:
After changing the type signature for `computeBakerStakesAndCapital`, this type application shouldn't be needed.

Reply:
yep, this has been changed.